torchdata/datapipes/iter/transform/bucketbatcher.py

```diff
@@ -181,13 +184,18 @@ def _default_len_fn(token):
     return len(token)
 
 
-def _token_len_fn(token, len_fn):
-    return len_fn(token), token
+@dataclass(order=True)
```

Comment:
```suggestion
@dataclass(order=True, frozen=True)
```
super nit: feel free to ignore, we can set `frozen` if we do not expect the need to modify it

Reply:
Makes sense.